### PR TITLE
Method to return parent of field

### PR DESCRIPTION
**Pull request by colmjude**

### CODE REVIEW

File: digital_land/specification.py [hunk 1]
```diff
@@ -147,6 +147,12 @@ def field_type(self, fieldname):
 
         raise ValueError("unknown datatype '%s' for '%s' field" % (datatype, fieldname))
 
+    def field_parent(self, fieldname):
+        field = self.field[fieldname]
+        if field["parent-field"]:
+            return field["parent-field"]
+        return ""
```
Comment (bram2000) [inline digital_land/specification.py:154]: I think I'd tend to return `None` in this case rather than empty string. Certainly not a rule but just feels better to mean "no result".

File: tests/unit/test_specification.py [hunk 2]
```diff
@@ -103,6 +103,11 @@ def test_field_typology():
     assert specification.field_typology("field-categories") == "category"
 
 
+def test_field_parent():
```
Comment (bram2000) [inline tests/unit/test_specification.py:106]: might as well add a test for a missing parent-field returning "" or None (depending on outcome of previous comment).

**Review by bram2000 — CHANGES_REQUESTED** [review]

File: digital_land/specification.py (hunk 1 above)
Comment (colmjude) [reply digital_land/specification.py:154]: I'd normally go with `None` too but noticed `""` had been used in the `field_typology` method [line 153](https://github.com/digital-land/digital-land-python/blob/main/digital_land/specification.py#L153)

Comment (colmjude) [reply digital_land/specification.py]: Sorted this with 43cfbd597

File: tests/unit/test_specification.py (hunk 2 above)
Comment (colmjude) [reply tests/unit/test_specification.py:106]: If the specification csv is loaded the `parent_field` will always exist so we decided a test for returning None isn't necessary.

File: digital_land/specification.py (hunk 1 above)
Comment (colmjude) [reply digital_land/specification.py]: And 8cbf333

**Review by bram2000 — APPROVED** [review]
